docs/content/specs-defs/includes/bicep/resource/non-functional/BCPRMNFR2.md

```diff
@@ -0,0 +1,46 @@
+---
+title: BCPRMNFR2 - User-defined types - AVM-Common-Types
+url: /spec/BCPRMNFR2
+geekdocNav: true
+geekdocAlign: left
+geekdocAnchor: true
+type: posts
+tags: [
+  Class-Resource, # MULTIPLE VALUES: this can be "Class-Resource" AND/OR "Class-Pattern" AND/OR "Class-Utility"
+  Type-NonFunctional, # SINGLE VALUE: this can be "Type-Functional" OR "Type-NonFunctional"
+  Category-Inputs/Outputs, # SINGLE VALUE: this can be "Category-Testing" OR "Category-Telemetry" OR "Category-Contribution/Support" OR "Category-Documentation" OR "Category-CodeStyle" OR "Category-Naming/Composition" OR "Category-Inputs/Outputs" OR "Category-Release/Publishing"  Language-Bicep, # MULTIPLE VALUES: this can be "Language-Bicep" AND/OR "Language-Terraform"
```

Comment:
Language-bicep tag looks commented out, please check

Reply:
This is by design as it is part of the comment. Hence also all the other key-words

Reply:
In the comment it says 'Single value' and the one that was chosen was `Category-Input/Outputs`